client/landing/stepper/declarative-flow/internals/steps-repository/blogger-starting-point/index.tsx

```diff
@@ -27,32 +28,35 @@ const StartingPointStep: Step = function StartingPointStep( { navigation } ) {
 	};
 
 	return (
-		<StepContainer
-			stepName="blogger-starting-point"
-			headerImageUrl={ startingPointImageUrl }
-			goBack={ goBack }
-			skipLabelText={ translate( 'Skip to dashboard' ) }
-			goNext={ () => submitIntent( 'skip-to-my-home' ) }
-			skipButtonAlign="top"
-			isHorizontalLayout={ true }
-			formattedHeader={
-				<FormattedHeader
-					id="intent-header"
-					headerText={ headerText }
-					subHeaderText={ subHeaderText }
-					align="left"
-				/>
-			}
-			stepContent={
-				<IntentScreen
-					intents={ intents }
-					intentsAlt={ [] }
-					onSelect={ submitIntent }
-					preventWidows={ preventWidows }
-				/>
-			}
-			recordTracksEvent={ recordTracksEvent }
-		/>
+		<>
+			<DocumentHead title={ translate( 'Blogger Starting Point' ) } />
```

Comment:
I'd use `Nice job! Now it’s time to get creative.` because that's the main header.
<img width="506" alt="Screen Shot 2565-12-08 at 18 03 14" src="https://user-images.githubusercontent.com/1881481/206430772-843d2c72-11ac-40b8-a607-6d5a078704fe.png">